omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -668,8 +668,14 @@
         "label": "Patient DSDM Model Report",
         "url": "reportingui/runReport.page?reportDefinition=941d0567-4848-41b2-8713-305667c89d01",
         "order": 125
+      },{
```

Comment:
@echiteri This needs to move to data exports 2019 since its not available in the previous versions

Reply:
@ssmusoke I don't understand this comment. The "Recency HTS Client Card Data Export" I added is under "UgandaEMR Reports - HMIS Tools 2019 Version" on reports already. Is that what you mean?